### PR TITLE
Added style for show event page

### PR DESCRIPTION
**Pull request by Adrianaito**
## General Description

Added some style for show events page, still needs some adjusts (choose font, color, etc)
Also made a card for the index page , just a suggestion. ✌️😃
Captura de Tela 2021-02-23 às 20 09 27

Captura de Tela 2021-02-23 às 20 09 43

## Fixed uploaded img size

Captura de Tela 2021-02-23 às 23 44 23

### CODE REVIEW

**Comment by SuzukiRyuichiro** [comment]
Screen Shot 2021-02-24 at 0 14 07
i love the new small sized cards. 
i just wanted to check that the index page is meant to be like this so we have to agree on which card to choose before merging right?

**Comment by SuzukiRyuichiro** [comment]
also, before merging, because Ken was also at the task of designing card in the index evnets, I think we have to make sure that there is no conflict in our merges. So I think we will discuss this tomorrow. Thank you so much for the suggestion thogh. I really love the elegant design <3

Also, I think we really ought to be sure with our wireframes tomorrow because I think we are lost with our design philosophies 😅

File: app/views/events/index.html.erb [hunk 1]
```diff
@@ -5,6 +5,15 @@
...
 <div class="container">
   <% @events.each do |event| %>
+    <div class="card card-index-sm" style="width: 18rem;">
+      <%= link_to " ", event_path(event), class: 'img-link' %>
+      <img src="https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/greece.jpg" />
+      <div class="card-body">
+      <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+      <img class="avatar-card avatar" src="https://kitt.lewagon.com/placeholder/users/krokrob" class="card-trip-user avatar-bordered" />
+    </div>
+</div>
```
Comment (SuzukiRyuichiro) [inline app/views/events/index.html.erb]: i believe this line needs a bit more indentation

**Comment by Adrianaito** [comment]
> also, before merging, because Ken was also at the task of designing card in the index evnets, I think we have to make sure that there is no conflict in our merges. So I think we will discuss this tomorrow. Thank you so much for the suggestion thogh. I really love the elegant design <3
> 
> Also, I think we really ought to be sure with our wireframes tomorrow because I think we are lost with our design philosophies 😅

yeah, sure it is just a suggestion and if we agree to merge it, there are still a few adjustments to be done! And I agree  that our design is a little messy ! But we'll get there! 💪😁

File: app/views/events/index.html.erb (hunk 1 above)
Comment (Adrianaito) [reply app/views/events/index.html.erb]: Got it ! ✌️

**Pull request merged**